src/desktop/components/react/stitch_components/NavBar.tsx

```diff
@@ -26,15 +27,17 @@ interface NavBarProps {
   user: SystemContextProps["user"]
   notificationCount: number
   searchQuery?: string
+  bannerCode?: string
```

Comment:
What does "code" refer to?

Reply:
totally arbitrary name - really it should match what we have in the eigen PR which is currently `confirmation_message`

Reply:
`confirmationMessage` 👍

Reply:
Also maybe `flashMessage` or something more generic that we can piggy back on for other things.

Reply:
We decided on `flashMessage` cc @brainbicycle @dleve123